src/components/sidebar/sidebar.component.js

```diff
@@ -10,10 +11,10 @@ const Sidebar = ({ editFormVisible, ...props }) => (
     `}
   >
     {
-      editFormVisible ? 
-        <EditForm {...props} />
+      useSelector(state => state.edit).visible ? 
```

Comment:
better to assign this to a variable to keep your template clean
```suggestion
      isVisible ? 
```

Reply:
Done